--- Sphinx-1.4.1/sphinx/application.py.orig	2016-05-12 20:44:55.230010066 +0200
+++ Sphinx-1.4.1/sphinx/application.py	2016-05-12 20:47:26.036670440 +0200
@@ -165,7 +165,7 @@
 
         # check the Sphinx version if requested
         if self.config.needs_sphinx and \
-           self.config.needs_sphinx > sphinx.__display_version__:
+           str(self.config.needs_sphinx) > sphinx.__display_version__:
             raise VersionRequirementError(
                 'This project needs at least Sphinx v%s and therefore cannot '
                 'be built with this version.' % self.config.needs_sphinx)
