--- Sphinx-1.7.6/setup.py.orig	2018-07-21 09:11:20.607543626 +0200
+++ Sphinx-1.7.6/setup.py	2018-07-21 09:14:00.757541787 +0200
@@ -42,7 +42,6 @@
         'whoosh>=2.0',
     ],
     'test': [
-        'mock',
         'pytest',
         'pytest-cov',
         'html5lib',
--- Sphinx-1.7.6/tests/test_build.py.orig	2018-07-21 09:38:14.234191868 +0200
+++ Sphinx-1.7.6/tests/test_build.py	2018-07-21 09:39:37.260857586 +0200
@@ -13,7 +13,10 @@
 import sys
 from textwrap import dedent
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 from docutils import nodes
 
--- Sphinx-1.7.6/tests/test_config.py.orig	2018-07-21 09:38:14.237525201 +0200
+++ Sphinx-1.7.6/tests/test_config.py	2018-07-21 09:39:59.077523735 +0200
@@ -9,7 +9,10 @@
     :copyright: Copyright 2007-2018 by the Sphinx team, see AUTHORS.
     :license: BSD, see LICENSE for details.
 """
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 from six import PY3, iteritems
 
--- Sphinx-1.7.6/tests/test_domain_std.py.orig	2018-07-21 09:38:14.237525201 +0200
+++ Sphinx-1.7.6/tests/test_domain_std.py	2018-07-21 09:40:23.530857064 +0200
@@ -9,7 +9,10 @@
     :license: BSD, see LICENSE for details.
 """
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from docutils import nodes
 
 from sphinx.domains.std import StandardDomain
--- Sphinx-1.5.5/tests/test_environment_indexentries.py.orig	2017-04-03 17:17:12.000000000 +0200
+++ Sphinx-1.5.5/tests/test_environment_indexentries.py	2017-04-16 19:16:37.202256421 +0200
@@ -13,7 +13,10 @@ from collections import namedtuple
 from sphinx import locale
 from sphinx.environment.managers.indexentries import IndexEntries
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 Environment = namedtuple('Environment', 'indexentries')
 
--- Sphinx-1.7.6/tests/test_ext_intersphinx.py.orig	2018-07-21 09:38:14.237525201 +0200
+++ Sphinx-1.7.6/tests/test_ext_intersphinx.py	2018-07-21 09:40:47.824190102 +0200
@@ -13,7 +13,10 @@
 import unittest
 from io import BytesIO
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import requests
 from docutils import nodes
--- Sphinx-1.7.6/tests/test_ext_napoleon_docstring.py.orig	2018-07-21 09:38:14.240858534 +0200
+++ Sphinx-1.7.6/tests/test_ext_napoleon_docstring.py	2018-07-21 09:41:38.977522865 +0200
@@ -15,7 +15,10 @@
 from textwrap import dedent
 from unittest import TestCase
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx.ext.napoleon import Config
 from sphinx.ext.napoleon.docstring import GoogleDocstring, NumpyDocstring
--- Sphinx-1.7.6/tests/test_ext_napoleon.py.orig	2018-07-21 09:38:14.240858534 +0200
+++ Sphinx-1.7.6/tests/test_ext_napoleon.py	2018-07-21 09:41:24.010856357 +0200
@@ -13,7 +13,10 @@
 from collections import namedtuple
 from unittest import TestCase
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from sphinx.application import Sphinx
 from sphinx.ext.napoleon import _process_docstring, _skip_member, Config, setup
--- Sphinx-1.5.5/tests/test_util_fileutil.py.orig	2017-03-27 03:42:15.000000000 +0200
+++ Sphinx-1.5.5/tests/test_util_fileutil.py	2017-04-16 19:17:19.555589270 +0200
@@ -11,7 +11,10 @@
 from sphinx.util.fileutil import copy_asset, copy_asset_file
 from sphinx.jinja2glue import BuiltinTemplateLoader
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 
 class DummyTemplateLoader(BuiltinTemplateLoader):
